appveyor.yml

```diff
@@ -6,7 +6,6 @@ environment:
   nodejs_version: "10"
 
 install:
-  - choco install googlechrome --ignore-checksums
```

Comment:
`choco` is having problems, see: https://github.com/angular/angular-cli/pull/9747/files#issue-171025810